app/i18n/translations.js

```diff
@@ -0,0 +1,7 @@
+import LocalizedStrings from 'react-native-localization';
+
+let strings = new LocalizedStrings({
+ "en-US":{
```

Comment:
@ggazzo We need to add the english text here. I saw that many of them are being updated at run time.